### PR TITLE
feat: badge component (for status and icons)

### PR DESCRIPTION
**Pull request by tushartiwari7**

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  margin: 2rem 1rem;
+  height: 50px;
+  line-height: 50px;
+}
+
+.badge-number {
+  position: absolute;
+  right: -5px;
+  top: -5px;
+  width: 15px;
```
Comment (Raghav888) [inline components/badge/badge.css:43]: Try to use rem here as it is more than 10px

**Review by Raghav888 — APPROVED** [review]
Looks good

**Comment by ultronjs** [comment]
Looks good

File: components/badge/badge.css (hunk 1 above)
Comment (tushartiwari7) [reply components/badge/badge.css:43]: Yeah sure! Thanks @Raghav888

**Pull request merged**